src/software/ai/hl/stp/play/free_kick/free_kick_play_fsm.cpp

```diff
@@ -0,0 +1,216 @@
+#include "free_kick_play_fsm.h"
+
+FreeKickPlayFSM::FreeKickPlayFSM(TbotsProto::AiConfig ai_config)
+    : ai_config(ai_config),
+      best_pass_and_score_so_far(
+          PassWithRating{.pass = Pass(Point(), Point(), 0), .rating = 0}),
+      align_to_ball_tactic(std::make_shared<MoveTactic>()),
+      shoot_tactic(std::make_shared<KickTactic>()),
+      chip_tactic(std::make_shared<ChipTactic>()),
+      passer_tactic(std::make_shared<KickTactic>()),
+      receiver_tactic(std::make_shared<ReceiverTactic>()),
+      offensive_positioning_tactics(std::vector<std::shared_ptr<MoveTactic>>(2)),
```

Comment:
Not blocking this merge but I think free kick play FSM is only handling two robot behaviour atm (can be fixed later) Or am I wrong?